functions.php

```diff
@@ -1243,9 +1245,8 @@ function validate_short_article_content_length($post_id)
 remove_filter('the_excerpt', 'wpautop'); // Remove <p> tags from Excerpt altogether
 remove_filter('the_content', 'wpautop');
 add_filter('the_content', 'wpautop', 12); //Remove <p> and <br> from shortcodes
-add_filter('use_block_editor_for_post_type', 'mongabay_disable_gutenberg_posts', 10, 2); //Enable Gutenberg editor for custom stories only
-add_filter('use_block_editor_for_post', 'mongabay_disable_gutenberg_pages', 10, 2); //Enable Gutenberg editor for certain pages only
+add_filter('use_block_editor_for_post', 'mongabay_disable_gutenberg', 10, 2); //Enable Gutenberg editor for certain pages and custom stories

```

Comment:
@Right-Noe are you sure having just use_block_editor_for_post filter is going to work on custom post types? I believe that's why use_block_editor_for_post_type filter was there in first place.